Library/src/Multiplayer/SpaceEntity.cpp

```diff
@@ -439,8 +465,8 @@ void SpaceEntity::SerialisePatch(IEntitySerialiser& Serialiser) const
 		Serialiser.WriteBool(false); // Destroy
 		Serialiser.BeginArray();	 // ParentId
 		{
-			Serialiser.WriteBool(false);
-			Serialiser.WriteNull();
+			Serialiser.WriteBool(ShouldUpdateParent);
```

Comment:
What effect does serializing this state have?

Reply:
Oh I see, it tells other connected CSP client applications that they too need to update the entity's parent 👍